### ISSUE TITLE
[suggestion] terrarian slimes compat

### ISSUE THREAD

**Issue by Chuunkyz** [PinkGoosik/visuality#8]
the slime effect doesnt apply to modded slimes, if you add this custom colors for the particles (related to the slimes color) would be really cool

**Comment by PinkGoosik** [comment]
Yep its planned, just need to came to a better implementation without hard depending on slime mods
